arelle/plugin/validate/DBA/rules/fr.py

```diff
@@ -50,6 +52,7 @@ def rule_fr1(
 
 @validation(
     hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=ALL_DISCLOSURE_SYSTEMS,
```

Comment:
I guess I wasn't very clear with my previous comment. `ALL_DISCLOSURE_SYSTEMS` is the default behavior, so you don't need to specify this. Just the rules which use `STAND_ALONE_DISCLOSURE_SYSTEMS` or `MULTI_TARGET_DISCLOSURE_SYSTEMS`.

Reply:
Ahhh, I knew that and for some reason completely forgot. Fixing now.